src/sass/lightbox-dialog/lightbox-dialog.scss

```diff
@@ -192,6 +192,21 @@ button.icon-btn.lightbox-dialog__prev {
         transform: rotate(180deg);
     }
 }
+
+@media (max-width: $_screen-size-SM) {
+    .lightbox-dialog--wide .lightbox-dialog__window {
```

Comment:
I think longer term we should be looking to make it not _full_ screen, but more like _almost_ full screen, so that we see a small area of mask around the dialog.